src/utils/data.js

```diff
@@ -1,3 +1,14 @@
+const KEY_DELIMETER = '.'
```

Comment:
I copied this stuff over from Microcosm. I have mixed feels on that, but I fully trust this code.